packages/@webex/plugin-meetings/src/rtcMetrics/index.ts

```diff
@@ -60,7 +64,18 @@ export default class RtcMetrics {
       if (data.name === 'stats-report') {
         data.payload = data.payload.map(this.anonymizeIp);
       }
+
       this.metricsQueue.push(data);
+
+      // If a connection fails, send the rest of the metrics in queue and get a new connection id.
+      if (
+        data.name === 'onconnectionstatechange' &&
+        data.payload[0] &&
```

Comment:
should we check for a type at data.payload? and then do data.payload?[0]

Reply:
Updated